packages/dolphin-emu-git/dolphin-emu-git.pacscript

```diff
@@ -0,0 +1,29 @@
+pkgname="dolphin-emu-git"
+gives="dolphin-emu"
+url='https://dolphin-emu.org'
+source=("${gives}::https://github.com/dolphin-emu/dolphin.git")
+makedepends=("build-essential" "ninja-build" "cmake" "ffmpeg" "libavcodec-dev" "libavformat-dev" "libavutil-dev" "libswscale-dev" "libevdev-dev" "libusb-1.0-0-dev" "libxrandr-dev" "libxi-dev" "libpangocairo-1.0-0" "qt6-base-private-dev" "libqt6svg6-dev" "libbluetooth-dev" "libasound2-dev" "libpulse-dev" "libgl1-mesa-dev" "libcurl4-openssl-dev" "libudev-dev" "libsystemd-dev | libeudev-dev")
+depends=("${makedepends[@]}")
+pkgver="5.0"
+maintainer=("Oren Klopfer <oren@taumoda.com>")
+pkgdesc='Dolphin is an emulator for running GameCube and Wii games'
```

Comment:
Don't include the name of the package as the first word. Maybe do `Emulator for running GameCube and Wii games`.